src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    controlled using an external failover coordinator.
+
+    In the `supervised` mode, `database.mode` and `<replicaset_name>.leader`
+    shouldn't be set explicitly.
+]])
+
+M['replication.peers'] = format_text([[
+    URIs of instances that constitute a replica set. These URIs are used by
+    an instance to connect to another instance as a replica.
+
+    Alternatively, you can use iproto.advertise.peer to specify a URI used to
+    advertise the current instance to other cluster members.
+]])
+
+M['replication.peers.*'] = format_text([[
+    specifies URIs of replica set instances.
+
+    For example: `- replicator:topsecret@127.0.0.1:3301`.
+]])
+
+M['replication.skip_conflict'] = format_text([[
+    By default, if a replica adds a unique key that another replica has added,
+    replication stops with the `ER_TUPLE_FOUND` error. If
+    `replication.skip_conflict` is set to `true`, such errors are ignored.
+]])
+
+M['replication.sync_lag'] = format_text([[
+    The maximum delay (in seconds) between the time when data is written to
+    the master and the time when it is written to a replica. If
+    `replication.sync_lag` is set to `nil` or `365 * 100 * 86400`
+    (`TIMEOUT_INFINITY`), a replica is always considered to be "synced".
+]])
+
+M['replication.sync_timeout'] = format_text([[
+    The timeout (in seconds) that a node waits when trying to sync with other
+    nodes in a replica set after connecting or during a configuration update.
+    This could fail indefinitely if `replication.sync_lag` is smaller than
+    network latency, or if the replica cannot keep pace with master updates.
+    If `replication.sync_timeout` expires, the replica enters `orphan` status.
+]])
+
+M['replication.synchro_queue_max_size'] = format_text([[
+    Puts a limit on the number of transactions in the master synchronous queue.
+
+    `replication.synchro_queue_max_size` is measured in number of bytes to
+    be written (0 means unlimited, which was the default behaviour before).
+    This option affects only the behavior of the master, and defaults to
+    16 megabytes.
+
+    Now that `replication.synchro_queue_max_size` is set on the master node,
+    tarantool will discard new transactions that try to queue after the limit
+    is reached. If a transaction had to be discarded, user will get an error
+    message "The synchronous transaction queue is full".
+
+    This limitation does not apply during the recovery process.
+]])
+
+M['replication.synchro_quorum'] = format_text([[
+    A number of replicas that should confirm the receipt of a synchronous
+    transaction before it can finish its commit.
+
+    This option supports dynamic evaluation of the quorum number. For example,
+    the default value is `N / 2 + 1` where `N` is the current number of replicas
+    registered in a cluster. Once any replicas are added or removed, the
+    expression is re-evaluated automatically.
```

Comment:
I pass it over, where it is obvious that 'cluster' means a replicaset. But here I would highlight it. Let's use 'replica set' instead of 'cluster'.

Reply:
Ok. Let's use `replica set` here.